practice-app/scholar_util.py

```diff
@@ -40,6 +40,8 @@ def get(self):
 
         name = parser.parse_args().get('name')
         _range = parser.parse_args().get('range')
+        _range = int(_range)

```

Comment:
Since range is not required, parse_args might return none and casting to integer might fail.

Reply:
I handled the None input case and other potentially problematic inputs.